gemfiles/rails_5_2.gemfile

```diff
@@ -36,3 +36,4 @@ gem "listen", "~> 3.0.5"
 gem "spring-watcher-listen", "~> 2.0.0"
 gem "pg", "~> 0.18"
 gem "sqlite3", "~> 1.3.6"
+gem "aasm"
```

Comment:
Not sure what the policy is about these kind of tests that require external libraries. I now only require this gem in the relevant spec but it's still needed in the gemfiles. I'm happy to change / remove it.